org.eclipse.osgitech.rest.jetty/src/main/java/org/eclipse/osgitech/rest/jetty/JettyServerRunnable.java

```diff
@@ -65,7 +65,7 @@ public void lifeCycleStopping(LifeCycle lifeCycle) {
 				
 				@Override
 				public void lifeCycleStopped(LifeCycle lifeCycle) {
-					state = JerseyServiceRuntime.State.STARTED;
+					state = JettyBackedWhiteboardComponent.State.STARTED;
```

Comment:
Is state STARTED really correct? 
The method has the name *lifeCycleStopped*, so I would expect a similar State STOPPED

Reply:
That's a good question. It's not something that I changed (other than the name of the enclosing type). I agree that it looks odd. I think it might be best to look at issue #12 to see if we can remove this completely and rely on the native Jersey implementation